src/metabase/models/permissions.clj

```diff
@@ -1053,6 +1084,12 @@
         (do
           (classloader/require 'metabase-enterprise.advanced-permissions.models.permissions)
           ((resolve 'metabase-enterprise.advanced-permissions.models.permissions/update-db-download-permissions!)
+           group-id db-id new-perms))
```

Comment:
I think we should add a check for if `ee` is available for `download` and `data-model`.
It's unlikely but it will return 500 if someone using OSS and they send a graph update that has `download` or `data-model`.

Reply:
Yeah, my logic here was that the frontend would never send changes to EE perms if EE is not available. But you're right that someone could call it over the API. Will change.